index.html

```diff
@@ -171,12 +171,15 @@ <h2>Let's chat!</h2>
         <form action="https://formspree.io/f/mbjqaodr" method="POST" class="container">
           <ul>
             <li>
-              <input type="text" id="name" name="user_name" placeholder="Text Input" maxlength="30" required />
+              <label for="name">name</label>
+              <input type="text" id="name" name="user_name" placeholder="Alex Gamino" maxlength="30" required />
```

Comment:
You have the input correctly placed ahead of the others as the previous reviewer requested. However, the form overlaps the `Contact us` text. It would be better to remove the `margin-top: 39px` property on the `input` element since the `li` tags that wrap them already have a margin set on them for spacing. Kindly implement this before you merge. 👍🏽